youtube_dl/extractor/heise.py

```diff
@@ -54,6 +54,7 @@ class HeiseIE(InfoExtractor):
     }]
 
     def _real_extract(self, url):
+        url = url.replace('www', 'm', 1)
```

Comment:
No. If `www` happens to be in path, query or fragment of mobile URL this will produce broken mobile URL.